modules/aws/s3.go

```diff
@@ -478,6 +478,37 @@ func AssertS3BucketPolicyExistsE(t testing.TestingT, region string, bucketName s
 	return nil
 }
 
+// AssertS3BucketServerSideEncryption checks if the given S3 bucket has a server side encryption configured using the given algorithm and fail the test if it does not
+func AssertS3BucketServerSideEncryption(t testing.TestingT, region string, bucketName string, algorithm types.ServerSideEncryption) {
```

Comment:
Unused function 'AssertS3BucketServerSideEncryption' - may be a problem if at some point, the function is removed and there will not be any test that fails